interfaces/i18n.js

```diff
@@ -0,0 +1,18 @@
+type chrome$LanguageCode = string;
+
+type chrome$DetectedLanguage = {
+  language: chrome$LanguageCode,
+  percentage: number,
+};
+
+type chrome$LanguageDetectionResult = {
+  isReliable: boolean,
+  languages: Array<chrome$DetectedLanguage>,
+};
+
+type chrome$i18n = {
+  getAcceptLanguages(callback: (languages: Array<chrome$LanguageCode>) => void): void,
+  getMessage: (messageName: string, substitutions?: string | Array<string>) => string,
```

Comment:
Change this back to the method annotation style
```js
getMessage(...): type,
```

Reply:
Will do.